### ISSUE TITLE
I want to ask a small question and look forward to your answer.

### ISSUE THREAD

**Issue by yaqiongxian** [matteo-rizzo/fc4-pytorch#15]
Hello, I am a beginner who has just started to learn in this field. I recently ran your code and I have a small doubt. Could you please help me solve it? Why do the images output by the neural network have a red bias, while the gt_corrected images generated in the preprocessing process using [img2npy.py]are normal? Do I need to do some processing for the images generated by the network? Thank you for your help.

**Comment by matteo-rizzo** [comment]
Hello and thank you for your question! Unfortunately I'm not sure what you mean by "red bias". Could you please enlarge on this? Did you do any measurement or is it just an impression? The correction is carried out by the "correct" function in this file: https://github.com/matteo-rizzo/fc4-pytorch/blob/main/auxiliary/utils.py. Maybe looking into that could help you address your doubts. Let me know! :)

**Comment by yaqiongxian** [comment]
Dear matteo rizzo
           I'm very excited that you are able to reply to me, and appreciate your response very much.I'm sorry that my English is not very good, so it may have caused some ambiguity. This time I attached two pictures for illustration.
As you can see, Figure 1 is the correction image generated by inputting the true light source color label during network loss prediction, while Figure 2 is the correction image generated by inputting the same true label during preprocessing. Why do they have a big difference in visual effect? Is it because I missed something during the dataset establishment process? I'm looking forward to your response and thank you again for your patience and explanation!
















At 2023-05-29 16:56:51, "Matteo Rizzo" ***@***.***> wrote:

Hello and thank you for your question! Unfortunately I'm not sure what you mean by "red bias". Could you please enlarge on this? Did you do any measurement or is it just an impression? The correction is carried out by the "correct" function in this file: https://github.com/matteo-rizzo/fc4-pytorch/blob/main/auxiliary/utils.py. Maybe looking into that could help you address your doubts. Let me know! :)

**Comment by matteo-rizzo** [comment]
Hi again! I'm sorry but it seems like I'm unable to see the pictures you wanted to show. Could you please try to send them again? Otherwise feel free to reach out via email

**Comment by yaqiongxian** [comment]
Thank you very much for your reply！ I have already sent my picture to your personal email. :D
